SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -309,6 +309,14 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, Config, Category = "Interest", meta = (EditCondition = "bEnableNetCullDistanceFrequency"))
 	TArray<FDistanceFrequencyPair> InterestRangeFrequencyPairs;
 
+	/** Use TLS encryption for UnrealClient workers connection. May impact performance. */
```

Comment:
I wonder if all's gonna be fine in case where the server and the client have different encryption option?

Reply:
It will, this is worker to runtime.

Reply:
👍